localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -164,9 +164,14 @@ NDTScanMatcher::NDTScanMatcher()
     "ekf_pose_with_covariance", 100,
     std::bind(&NDTScanMatcher::callback_initial_pose, this, std::placeholders::_1),
     initial_pose_sub_opt);
-  sensor_points_sub_ = this->create_subscription<sensor_msgs::msg::PointCloud2>(
-    "points_raw", rclcpp::SensorDataQoS().keep_last(points_queue_size),
-    std::bind(&NDTScanMatcher::callback_sensor_points, this, std::placeholders::_1), main_sub_opt);
+  sensor_points_sub_ = std::make_shared<message_filters::Subscriber<sensor_msgs::msg::PointCloud2>>(
```

Comment:
(Not related to the code review!)

Do you have any evaluation results on `no_ground_*` scores? I understood that the data posted on the previous issue (https://github.com/autowarefoundation/autoware.universe/issues/2044) is a bit different from this PR's algorithm (since it is performing NDT using scans without ground removal). It would be great if you would share with us some of your results to show that this change is reasonable enough!

IMHO posting and discussing the evaluation report in this PR is a bit inconvenient, so I propose to open a new [GitHub Discussion](https://github.com/orgs/autowarefoundation/discussions) for this topic :smile:

Reply:
share evaluation results, [discussion3056](https://github.com/orgs/autowarefoundation/discussions/3056)

Reply:
Thank you!